src/server/config.ts

```diff
@@ -4,6 +4,13 @@ export const TEST_ENV = !!(process.env.CI || process.env.NODE_ENV === 'test')
 
 export const PROD_ENV = process.env.NODE_ENV === 'production'
 
-export const getUploadTmpDirectory = () => {
-    return process.env.UPLOAD_TMP_DIRECTORY || '/tmp'
-}
+export const getUploadTmpDirectory = () => process.env.UPLOAD_TMP_DIRECTORY || '/tmp'
+
+export const ALWAYS_CREATE_ECR = process.env.ALWAYS_CREATE_ECR === 't'
```

Comment:
just out of curiosity why are the env variables stored as `t` instead of true/false? something outside of our control / we cant use booleans?

Reply:
yes they come from the process.env and all values are strings.  I could have just tested for their existence, but this way they can also be turned off if needed by setting to anything other than `'t'`